src/resources/Catalogs/CatalogContent.ts

```diff
@@ -6,21 +6,11 @@ export type ObjectType = {
     objectType: string;
 };
 
-export type Version = {
-    version: number;
-};
-
 export default class CatalogContent extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/catalogcontent/source`;
 
-    getObjectTypeV2(sourceId: string, version: Version) {
-        return this.api.get<CatalogObjectType>(
-            this.buildPath(`${CatalogContent.baseUrl}/${sourceId}/objecttypes`, version),
-        );
-    }
-
-    getObjectTypes(sourceId: string) {
```

Comment:
Should this one be kept instead? Its implementation was already the right thing

Reply:
that works too, I will change the naming back to `getObjectTypes` to make it less confusing